stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/SheetViewModel.kt

```diff
@@ -50,6 +50,6 @@ internal abstract class SheetViewModel<TransitionTargetType, ViewStateType>(
     }
 
     fun updateSelection(selection: PaymentSelection?) {
-        mutableSelection.postValue(selection)
```

Comment:
why this change?  are we guaranteed that updateSelection will only ever be called from the UI thread?

Reply:
Yeah, I verified that this is always called from the UI thread.